src/tests/ftest/container/full_pool_container_create.py

```diff
@@ -7,8 +7,36 @@
 import time
 
 from apricot import TestWithServers
-from general_utils import DaosTestError, get_random_bytes
-from test_utils_container import TestContainerData
+from general_utils import DaosTestError
+
+DER_NOSPACE = "-1007"
+
+
+def fill_container(self, container):
```

Comment:
I think it would be better if this didn't need `self`. We could just raise an exception and let the caller decide whether it's fatal

Reply:
Addressed